src/introduction.md

```diff
@@ -3,37 +3,117 @@
 All claims, content, designs, algorithms, estimates, roadmaps, specifications,
 and performance measurements described in this project are done with the
 author's best effort. It is up to the reader to check and validate their
-accuracy and truthfulness. Furthermore nothing in this project constitutes a
+accuracy and truthfulness. Furthermore, nothing in this project constitutes a
 solicitation for investment.
 
-# Introduction
+# What is Solana?
 
-This document defines the architecture of Solana, a blockchain built from the
+Solana is the name of an open source project that is implementing a new
```

Comment:
recommend striking "new" and changing to active voice: "that implements a blockchain"